net/lagrange/Portfile

```diff
@@ -9,7 +9,7 @@ PortGroup           compiler_blacklist_versions 1.0
 # clock_gettime
 legacysupport.newest_darwin_requires_legacy 15
 
-github.setup        skyjake lagrange 1.8.0 v
+github.setup        skyjake lagrange 1.8.2 v
 revision            1
```

Comment:
please reset the `revision` when updating the `version`

Reply:
Fixed, always forget about revision.